DownFall/Views/Button.swift

```diff
@@ -71,13 +82,17 @@ class Button: SKSpriteNode {
          identifier: ButtonIdentifier,
          precedence: Precedence,
          fontSize: CGFloat,
-         fontColor: UIColor) {
+         fontColor: UIColor,
+         backgroundColor: UIColor = .menuPurple) {
```

Comment:
Do we really use the other init method that doesn't take a identifier?  It makes life harder that indentifier is optional